examples/tutorials/nuxt.md

```diff
@@ -0,0 +1,423 @@
+);
+</script>
+
+<template>
+  <main v-if="dinosaur">
+    <h1 class="text-2xl font-bold mb-4">{{ dinosaur.name }}</h1>
+    <p class="mb-4">{{ dinosaur.description }}</p>
+    <NuxtLink to="/" class="text-blue-600 hover:text-blue-800 hover:underline">
+      Back to all dinosaurs
+    </NuxtLink>
+  </main>
+</template>
+```
+
+Next, we’ll have to connect these Vue components together so that they render
+properly when we visit the root of the domain. Let’s update `app.vue` at the
+root of the directory to serve our application’s root component. We’ll use
+[`NuxtLayout`](https://nuxt.com/docs/api/components/nuxt-layout) for consistent
+page structure and [`NuxtPage`](https://nuxt.com/docs/api/components/nuxt-page)
+for dynamic page rendering:
+
+```tsx
+// app.vue
+
+<template>
+  <NuxtLayout>
+    <div>
+      <nav class="p-4 bg-gray-100">
+        <NuxtLink to="/" class="text-blue-600 hover:text-blue-800">
+          Dinosaur Encyclopedia
+        </NuxtLink>
+      </nav>
+
+      <div class="container mx-auto p-4">
+        <NuxtPage />
+      </div>
+    </div>
+  </NuxtLayout>
+</template>;
+```
+
+Let’s run our server with `deno task dev` and see how it looks at
+`localhost:3000`:
+
+<figure>
+
+<video class="w-full" alt="Build a Nuxt app with Deno." autoplay muted loop playsinline src="./images/how-to/nuxt/nuxt-3.mp4"></video>
+
+</figure>
+
+Looks great!
+
+## Add Tailwind
+
+Like we said, we're going to add a little bit of styling to this application.
+First, we'll set up a layout which will provide a consistent structure across
+all pages using Nuxt's layout system with
+[slot-based](https://vuejs.org/guide/components/slots) content injection:
+
+```tsx
+// layouts/default.vue
+
+<template>
+  <div>
+    <slot />
+  </div>
+</template>;
+```
+
+In this project, we’re also going to use [tailwind](https://tailwindcss.com/)
+for some basic design, so we need to install those dependencies:
+
+```bash
+deno install -D npm:tailwindcss npm:postcss npm:autoprefixer
+```
+
+Then initialize Tailwind:
+
+```bash
+deno -A npm:tailwindcss init
+```
+
+This will create a `tailwind.config.js` file in the root of our directory. Let’s
+open that up and update the `content` section to include key files in our
+project:
+
+```tsx
+// tailwind.config.js
+
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: [
+    "./components/**/*.{js,vue,ts}",
+    "./layouts/**/*.vue",
+    "./pages/**/*.vue",
+    "./plugins/**/*.{js,ts}",
+    "./app.vue",
+  ],
+  theme: {
+    extend: {},
+  },
+  plugins: [],
+};
+```
+
+Next, let’s add the tailwind utilities to a new css file, `assets/css/main.css`:
+
+```tsx
+// assets/css/main.css
+
+base;
+components;
+utilities;
+```
+
+The only other thing we'll need to do is update our `nuxt.config.ts` file to
+configure our Nuxt application for Deno compatibility, enable development tools,
+and set up Tailwind CSS. We're disabling SSR for better Deno compatibility and
```

Comment:
Is there a way to phrase this that implies less of a limitation for Deno? It sounds a but like Deno can't do SSR. It might be a touch ambiguous if we mean the runtime or Deno Deploy when we say that too.

Maybe...

```suggestion
and set up Tailwind CSS. We're disabling SSR and instead generating the entire site for the greatest flexibility in deployment options, and
```